db/schema.rb

```diff
@@ -42,6 +42,8 @@
   create_table "events_users", id: false, force: :cascade do |t|
     t.integer "event_id", null: false
     t.integer "user_id", null: false
+    t.index ["event_id", "user_id"], name: "index_events_users_on_event_id_and_user_id"
+    t.index ["user_id", "event_id"], name: "index_events_users_on_user_id_and_event_id"
```

Comment:
Is having an index important here? And if yes, do you mind explaining to me why?^^

Reply:
This just resulted from merging code from the dev-branch. I also don't know why they've added it. What do you suggest?